docs/document/docs/components/desktop/table.md

```diff
@@ -6,5 +6,133 @@ original_id: table
 ---
 
 ## Overview
+The Table component allows the user to display an editable table.
 
-Table English language
+<div class="sample-container" id="table">
+  <div id="sample-container__components"></div>
+</div>
+<script src="/js/samples/desktop/table.js"></script>
+
+---
+
+## Specification
+
+### Property
+
+Here is a list of properties that can be used for modifying the component:
+
+#### Table
+
+| Name | Type | Default | Description | Remark |
+| :--- | :--- | :--- | :--- | :--- |
+| className | string | ""  | Component class name | |
+| id | string | "" | Component id name | |
+| label | string | ""  | Label for the component | Label will not be displayed if unspecified or left empty |
+| actionButton | boolean | true | Show/Hide the add/remove row button | |
+| visible | boolean | true | Show/Hide the component | |
+| columns | Array\<[Columns](#column)\> |  []  | Column data of the component | Throw an error if the columns is not an array. |
+| data | Array\<object\> | []  | Row data of the component | Throw an error if the data is not an array. |
+
+#### Column
```

Comment:
How about writing inside "### Property" like this?
Column.field, Column.title, etc.
https://kintone-ui-component.netlify.app/docs/en/components/desktop/dropdown

Reply:
I think it's better to separate as it is now. Because column properties are quite complicated. So we should separate which properties are of table and column.
When the user looks at the Table's document, they can see that the table has 7 properties and the column has 5 properties.

Reply:
Thanks for the opinion! I agreed. Let's keep the current one.